scripts/dts/edtsdatabase.py

```diff
@@ -0,0 +1,217 @@
+#
```

Comment:
This commit was introduced as a cut out of one of your initial commits (https://github.com/zephyrproject-rtos/zephyr/commit/1d6fa538d45a4fcc95902122b97cda195bf33511 didn't exist at that time), I take care to keep your signed-off and copyright.
Few changes have been done in order to fit with modifications done inside this PR.
Using commit https://github.com/zephyrproject-rtos/zephyr/commit/1d6fa538d45a4fcc95902122b97cda195bf33511 will require to amend it to fit current PR anyway. Am I missing something?

Reply:
I would really be wiling to help, but since edts review basically happens here, I do the changes in EDTS class here. Taking commits from #6762 would require for you to do the changes and then I'd need to take them, push here, see if accepted, iterate.. Process might be a bit complex and slow, don't you think?

Reply:
The process I was asking for is for you to do a last time cherry-pick of  1d6fa53  into #9876. From there on I will rebase #6762 on #9876. All development will happen in #9876 and #6762 will be rebased from time to time until #9876 is merged into master.

Reply:
Just saw you comment, but did it before hand.
FYI I also fixed a small issue in device-types filling, so you might want to rebase again (or wait review completes)